docs/articles/opinion/Self-Governance_as_a_Human_Right.md

```diff
@@ -0,0 +1,155 @@
+# Self-Governance as a Human Right
+
+_Author [Marik Hazan / @marikio](https://github.com/marikio)_
+
+*(Kamaria’s name has been changed for her safety)*
+
+She was astounded no one noticed her.
+
+White skin, dressed in all black. Rash guard, board shorts, and a burn bag with a first aid kit, water shoes, and fins. Sweating from the anxiety, Kamaria knew that the trip she’d make in several hours could end in her throat being cut by smugglers, or police kidnapping her again.
+
+They were in a tourist district, not too far from the hotels. Moms held kids and rucksacks, while dads pointed cameras out the bus windows taking shots of Seljuk buildings and mustached Dondurma vendors tricking tourists into reaching for their waffle cones.
+
+A few months earlier a former Turkish military man had shown her the site where she was headed. He had done so as a joke, not thinking that she would ever end up coming back. At the time, Kamaria didn’t either. It was only a while later that she realized it was the perfect spot to stand watch.
+
+Several hours after the bus had dropped her off, she headed towards the long pier, about a kilometer away from the closest hotel. She had to wait another 6 to 7 hours before the boats would start leaving port. Watching with binoculars pressed tightly to her eyes, she would count the number of boats to undock and later warn her contacts on the Greece coastline of how many refugees they should expect.
```

Comment:
maybe this is not too important but although this intro drags the reader in, I kinda miss a clearer link with the article, was this supposed to come back? Now I want to know more about Kamaria - is she working for the greek government? Maybe just me but this part made me wonder

Reply:
Hmm, ok. I will try to link it back in towards the end.